tests/01-test_argument_parsing.sh

```diff
@@ -30,7 +30,7 @@ check_run_ok() {
 }
 
 #Check wrong combinations
-check_output_fail 'ansible-deploy' '\[ERROR\]: Too few arguments'
+check_output_fail 'ansible-deploy' 'Too few arguments'
```

Comment:
Maybe some new log-level for early significant errors e.g. fatal.
[FATAL]: Too few arguments!
;)

Reply:
it may make sense, if we use it in logger for all the errors where we exit, but... logging doesn't have it. It has a critical which sounds like somthing we should us for critical errors already - diferent issue.